### ISSUE TITLE
Define google_product_category per product

### ISSUE THREAD

**Issue by icemansparks** [dim00z/gshoppingflux#90]
First of all let me tell you that the module seems to be working fine on Prestashop 1.7.6.7.
All the settings and export functions work as expected.

Unfortunately i came across a problem and i don't know how to handle it correctly.

The categories in my shop are created thematically and thus one category can hold multiple products that don't really belong to the same google product category or even have the same attribute requirements.

I didn't find any solution yet how i could define the values based on each single product - is this even a valid approach?

In google merchant center i can define a second feed (e.g. google doc) to just manually update all the category assignments for each product / variant id, but this is tedious and slow for a large amount of variants.
While in facebook i can only directly import the xml file that was generated with the modules export function, and this leads to wrong assignments for the products added to facebook.

TL:DR
Would there be a possibility to assign each product to its specific google product category that can be used for the export when its set? - otherwise use the one from the category assigns.

**Comment by Casper-O** [comment]
Product Category is one thing, that gives a lot of headache for users of this module and is one of the top questions in my DM.

If i add the option to make it per product also, it would add an other level of "why is it not exporting correctly".

Maybe in a future rewrite of the module, but not for the current
